internal/iso3166/iso3166.go

```diff
@@ -76,7 +76,7 @@ var countryCodes = map[string]bool{
 	"CI": true, // Côte d'Ivoire
 	"HR": true, // Croatia
 	"CU": true, // Cuba
-	"CW": true, // Curaçao
+	"CW": true, // CuraCao
```

Comment:
incorrect replacement

Reply:
These files are created by `make generate` from an external source. I'll update it.